pytest_html/resources/main.js

```diff
@@ -12,18 +18,84 @@ function toArray(iter) {
 
 function find(selector, elem) {
     if (!elem) {
-        elem = document;
+        elem = DOCUMENT;
     }
     return elem.querySelector(selector);
 }
 
 function find_all(selector, elem) {
     if (!elem) {
-        elem = document;
+        elem = DOCUMENT;
     }
     return toArray(elem.querySelectorAll(selector));
 }
 
+function reset_sort_headers() {
```

Comment:
The patch is hard to read because these functions have been moved - could you move them back so the diff shows the lines changed more clearly?